src/Payload.h

```diff
@@ -34,6 +34,9 @@ struct Payload {
   std::string cloneDataToString() const;
   void clear();
 
+  // Clone this object onto the stack
```

Comment:
Not sure the `onto the stack` part is helpful.  A Payload is a tuple of 2 heap pointers, might be confusing.  No good ideas on what to say instead though...